lib/std/crypto/aes_cmac.zig

```diff
@@ -0,0 +1,170 @@
+/// RFC 4493 - The AES-CMAC Algorithm
+/// https://www.rfc-editor.org/rfc/rfc4493
+const std = @import("std");
+const Aes128 = std.crypto.core.aes.Aes128;
+
+pub fn AesCmac() type {
+    return struct {
+        const Self = @This();
+        var ctx: std.crypto.core.aes.AesEncryptCtx(Aes128) = undefined;
+
+        /// Create a new cmac context with the given key.
+        pub fn init(key: [16]u8) Self {
+            ctx = Aes128.initEnc(key);
+            return Self{};
+        }
+
+        /// Generates a cmac from the given message.
+        pub fn generate(self: Self, msg: []const u8) [16]u8 {
+            _ = self;
```

Comment:
This discard is incorrect. Zig prevented you from accidentally using global variables, and this line is incorrectly disabling that check.
```suggestion
```

Reply:
I have changed that and it should be ok now.